File: New/Hacker Rank/Assignment 1/py/Scripts/Cats and a Mouse.py
#Cats and a Mouse
def catAndMouse(x,y,z):
    difx=z-x
    dify=z-y
    if difx<0:
        difx=-difx
    if dify<0:
        dify=-dify
    if dify==difx:
        return "Mouse C"
    elif dify>difx:
        return "Cat A"
    else:
        return "Cat B"

q=int(input())
if q in range(1,100):
    for i in range(q):
        x, y, z = int(input()), int(input()), int(input())
        print(catAndMouse(x,y,z))

File: New/Hacker Rank/Assignment 1/cpp/Cats and a Mouse.cpp
//Cats and a Mouse
#include<iostream>
using namespace std;
string catAndMouse(int x,int y,int z)
{
    int difx=0,dify=0,win=0;
    difx=z-x;
    dify=z-y;
    if(difx<0)
    {
        difx=-difx;
    }
    if(dify<0)
    {
        dify=-dify;
    }
    if(difx==dify)
    {
        return "Mouse C";
    }
    else if(dify>difx)
    {
        return "Cat A";
    }
    else
    {
        return "Cat B";
    }

}
int main()
{
    int q=0,x=0,y=0,z=0;
    cin>>q;
    if(q>=1 && q<=100)
    {
        for(int i=0;i<q;i++)
        {
            cin>>x>>y>>z;
            cout<<catAndMouse(x,y,z)<<endl;
        }
    }
    return 0;
}
